Just to check the environment is set up okay

In [1]:
import numpy as np
import matplotlib.pyplot as plt
%matplotlib inline
import tensorflow as tf
print(tf.__version__)

2.11.0


Version should be 2.11.x.  If you get some warnings about not using/being optimised for a GPU don't worry it may well still run fine.